doc/bitcoin-conf.md

```diff
@@ -30,6 +30,21 @@ Network specific options can be:
 - placed into sections with headers `[main]` (not `[mainnet]`), `[test]` (not `[testnet]`) or `[regtest]`;
 - prefixed with a chain name; e.g., `regtest.maxmempool=100`.
 
+Network specific options take precedence over non-network specific options.
```

Comment:
Perhaps an example of network-specific vs non network-specific would be helpful. For example, IIUC the first instance of `rpcport` in the example below is not network-specific and the two instances after are network-specific, and so the first instance of the second two prevails. The example could take the form of inline code comments in the example below.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/16448#discussion_r306688348

> Perhaps an example of network-specific vs non network-specific would be helpful. 

Best to review the document as a whole and suggest specific changes in the appropriate sections:

https://github.com/fanquake/bitcoin/blob/options_bitocin_conf/doc/bitcoin-conf.md

I agree inline code comments below could be a good idea if you want to suggest them inside the example. But the paragraph immediately before this diff explains what network specific options are, so if that section is not clear we should fix it instead of trying to explain network options a second time at the same time as describing precedence.

Reply:
Thank you @ryanofsky, agreed, I made the mistake of overlooking reading the preceding paragraph.